core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -895,6 +897,17 @@ void populate1() {
       defineMethod(TYPEOF, BuiltInMethod.TYPEOF.method, NullPolicy.STRICT);
       defineMethod(VARIANTNULL, BuiltInMethod.VARIANTNULL.method, NullPolicy.STRICT);
 
+      // Left shift operations: shift bits to the left by specified amount.
+      // Supports integer, unsigned integer, and binary data types.
+      // Shift amount is normalized using modulo arithmetic based on data type bit width.
+      //
+      // "<<" operator: Infix syntax for bitwise left shift in SQL expressions (e.g., x << y)
```

Comment:
if you will add a unit test, maybe you can split this comments into 2 parts, where each function is documented separately